src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -558,7 +558,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
   Targets related to creating .zip/.tar.gz
   #########################################
   -->
-  <Target Name="_DownloadAndExtractDotNetRuntime" Condition="'$(DotNetBuildFromSource)' != 'true'">
+  <Target Name="_DownloadAndExtractDotNetRuntime" Condition="'$(DotNetBuild)' != 'true'">
```

Comment:
These archives are available in the VMR, just on the local path. Perhaps that's worth using?

Reply:
Could save us a minute or so in download time, where are they located?

Reply:
They don't get passed into aspnetcore yet. Surprising. But for installer it gets passed in as "PublicBaseUrl": https://github.com/dotnet/installer/blob/main/src/SourceBuild/content/repo-projects/installer.proj#L33

aspnetcore core will eventually need this or it won't find the runtime artifacts necessary to produce the hosting bundle.